xml_loader/demand.py

```diff
@@ -0,0 +1,47 @@
+from datetime import time, datetime, date
+class Demand():
```

Comment:
I think these files are moved to the folder `preprocessing` now. We should only have one implementation of the same concept, so it would be really nice if you could remove one of them

Reply:
I think this is the same as the other files that doesn't make sense. They are old files and should not be included in the merge.